tests/test_finder.py

```diff
@@ -401,6 +401,44 @@ def test_with_sdkconfig_defaults_env_var_expansion(self, tmp_path, monkeypatch):
         apps = find_apps('test1', 'esp32', recursive=True, config_rules_str='sdkconfig.ci.*=')
         assert len(apps) == 0
 
+    def test_with_sdkconfig_override(self, tmp_path):
+        from idf_build_apps.build_job import (
+            BuildAppJob,
+        )
+        from idf_build_apps.main import (
+            setup_build_app_job_override_sdkconfig,
+        )
+
+        create_project('test1', tmp_path)
+        (tmp_path / 'test1' / 'sdkconfig.defaults').write_text(
+            'CONFIG_IDF_TARGET=esp32s2\nCONFIG_FREERTOS_IDLE_TASK_STACKSIZE=1516', encoding='utf8'
```

Comment:
```suggestion
'''
CONFIG_IDF_TARGET=esp32s2
CONFIG_FREERTOS_IDLE_TASK_STACKSIZE=1516
''', encoding='utf8'
```
nitpick, this looks nicer

Reply:
Updated